packages/cfd/src/sass/mt5-passwordmodal.scss

```diff
@@ -0,0 +1,226 @@
+.dc-modal__container_cfd-password-modal,
```

Comment:
@Dhruv-deriv lets move this file inside mt5-password-modal folder

Reply:
Done 👍